Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/SonarQubeCategoriseProjectTests.cs

```diff
@@ -42,7 +44,122 @@ public void SimpleProject_NoTestMarkers_IsNotATestProject()
             var result = BuildAndRunTarget("foo.proj", "");
 
             // Assert
-            result.AssertExpectedCapturedPropertyValue("SonarQubeTestProject", "False");
+            AssertIsNotTestProject(result);
+        }
+
+        [TestMethod]
+        [TestCategory("ProjectInfo")]
+        [TestCategory("IsTest")]
+        public void ExplicitMarking_IsTrue()
```

Comment:
Most of these tests were moved from _WriteProjectInfoFileTargetTests.cs_. I'll call out the couple of new tests I've added.